ci/do_ci.sh

```diff
@@ -210,6 +221,7 @@ elif [[ "$CI_TARGET" == "bazel.asan" ]]; then
     pushd "${ENVOY_FILTER_EXAMPLE_SRCDIR}"
     bazel_with_collection test ${BAZEL_BUILD_OPTIONS} ${ENVOY_FILTER_EXAMPLE_TESTS}
     popd
+    run_process_test_result
```

Comment:
Can't this in bazel_with_collection function? It should work, no?

Reply:
When pushd into the filer_example_dir, it won't have access to the needed script. So this is me trying to get around that and do 
```
call script || echo "cannot find the script"
```
to prevent the program from failing when not being able to find the script.

Reply:
@lizan

Reply:
I think you can get original directory from $ENVOY_SRCDIR ?

Reply:
Let me try that quickly.